GiftedMessenger.js

```diff
@@ -647,6 +669,7 @@ GiftedMessenger.defaultProps = {
 
 GiftedMessenger.propTypes = {
   autoFocus: React.PropTypes.bool,
+  backgroundImage: React.PropTypes.number,
```

Comment:
This should be the imported image, so you should give th GiftedMessenger a prop like:
`backgroundImage={ require("image.png")`